sbc_compassion/tools/patternrecognition.py

```diff
@@ -167,7 +167,7 @@ def keyPointCenter(keypoints):
         return center / N
 
 
-def find_template(img, templates):
+def find_template(img, templates, threshold=4):
```

Comment:
Why did you add `threshold` if you don't use it inside the function ? Also update the method comment to add description of the parameter if you keep it.